sdcm/kafka/kafaka_cluster.py

```diff
@@ -0,0 +1,158 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2023 ScyllaDB
+
+import logging
+from pathlib import Path
+from functools import cached_property
+
+import requests
+
+from sdcm import cluster
+from sdcm.wait import wait_for
+from sdcm.remote import LOCALRUNNER
+from sdcm.utils.git import clone_repo
+from sdcm.utils.common import get_sct_root_path
+from sdcm.utils.remote_logger import CommandClusterLoggerBase
+
+# TODO: write/think more about the consumers
+
+logger = logging.getLogger(__name__)
+
+
+class KafkaLogger(CommandClusterLoggerBase):
+    # pylint: disable=invalid-overridden-method
+    @cached_property
+    def _logger_cmd(self) -> str:
+        return f"{self._cluster.compose_context} logs --no-color --tail=1000 >>{self._target_log_file}"
+
+
+class LocalKafkaCluster(cluster.BaseCluster):
+    def __init__(self, remoter=LOCALRUNNER):
+        super().__init__(cluster_prefix='kafka', add_nodes=False)
+        self.remoter = remoter
+        self.docker_compose_path = (
+            Path(get_sct_root_path()) / "kafka-stack-docker-compose"
+        )
+        self._journal_thread: KafkaLogger | None = None
+        self.init_repository()
+
+    def init_repository(self):
+        # TODO: make the url configurable
+        # TODO: create a fork/branch and remove default connector installers
+        # TODO: switch to specific branch/sha
+        # TODO: get the version after install, and send out to Argus
+        clone_repo(
```

Comment:
What about pinning version?

Reply:
No, I mean to pull `specific` version instead of the `latest/current`

Reply:
I've read it printing.... 

yeah we should pin it, and I want to fork it first, to remove a few things to make it start faster (it install 2 connector at the beginning that we don't need)

Reply:
I've now pinned it, to a fork.

later we'll move it to scylladb org, and start doing versions